packages/desktop-client/src/components/common/Text.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import { css } from 'glamor';
+import type { CSSProperties } from 'glamor';
+
+interface TextProps extends Omit<React.HTMLProps<HTMLSpanElement>, 'style'> {
+  style?: CSSProperties;
+  innerRef?: React.Ref<HTMLSpanElement>;
+  className?: string;
+  children?: React.ReactNode;
+}
+
+export const Text: React.FC<TextProps> = props => {
+  const { style, innerRef, ...restProps } = props;
+  return (
+    <span
+      {...restProps}
+      ref={innerRef}
+      className={`${props.className || ''} ${css(props.style)}`}
```

Comment:
Only because we ought to have similar utilities in emotion, we could lean into glamor a bit more:
```suggestion
      className={css(props.className, props.style)}
```

Reply:
(Fully aware this is old code, it just stood out to me.)

Reply:
This won't work: 
```
Argument of type 'string' is not assignable to parameter of type 'Rule'.
```

Reply:
We can put them in an array though. And then it works :)

Reply:
Nice. I tried it locally before commenting—away from the computer but maybe it was spreading the `css` return?